runelite-client/src/main/java/net/runelite/client/plugins/worldmap/WorldMapPlugin.java

```diff
@@ -301,7 +303,17 @@ private void updateShownIcons()
 				}
 			}).map(TeleportPoint::new)
 			.forEach(worldMapPointManager::add);
-	}
+
+			worldMapPointManager.removeIf(RunecraftingAltarPoint.class::isInstance);
+
```

Comment:
This blank line can be removed, as clearing RC altar points is directly related to the subsequent adding of them.
```suggestion
```